src/text-input/templates/aif/text-input.html

```diff
@@ -0,0 +1,9 @@
+<div class="text-input-wrapper">
+    <input
+        class="text-input"
+        id="{{ fieldId }}"
+        type="text"
+        ng-model="model"
+        ng-disabled="disable"
+        >
```

Comment:
Not really sure that having the `text-input` gives us anything but you might as well keep it